### PR TITLE
Feature for Bootcamp Git assignment

### PR DESCRIPTION
**Pull request by JanisRancans**
Bash script for Git & Bash assignment. The script creates new empty Bash script templates, adds shebang and makes them executable. Script must be run with two command line arguments - number of files to be created (no more than 20) and filename.

### CODE REVIEW

File: JanisRancans/bulk_create.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+if [ "$counter" -gt "20" ] || [ -z $filename ] || ! [[ "$counter" =~ ^[0-9]+$ ]];
+then
+        echo "Please provide correct args:"
+        echo "Number must be smaller than 20"
+        echo "Must have name"
+        echo "Example: ./bulk_create.sh 10 name"  
+else
+	for i in $(seq 0 $counter);
+        do
+		bash_location=$(which bash)
```
Comment (abuharis) [inline JanisRancans/bulk_create.sh:18]: Can we just add a conditional statement if something matches in while loop.

**Review by abuharis — CHANGES_REQUESTED** [review]
Its phenomenal, I'm not as good your scripts.
To understand the integration manager workflow. I will request for minor changes.

**Comment by JanisRancans** [comment]
Added requested changes and refactored code

**Comment by JanisRancans** [comment]
Added requested changes and refactored code

**Pull request merged**